java/shared/src/main/java/com/squareup/subzero/shared/SubzeroUtils.java

```diff
@@ -64,6 +67,15 @@ public class SubzeroUtils {
   static final String ERROR_INVALID_DESTINATION = "Invalid destination in output";
   static final String ERROR_INCONSISTENT_IS_CHANGE = "Inconsistent isChange in output";
 
+  /**
+   * Derives a P2SH-P2WSH address.
+   * @param network org.bitcoinj.params.TestNet3Params or org.bitcoinj.params.MainNetParams
+   * @param threshold Number of signatures needed to authorize a transaction.
+   * @param publicRootKeys The public keys from wallet initialization
+   * @param path The HD wallet Path.
```

Comment:
minor nit, sometimes there is `.` at the end of the comment but sometimes not. is there an intentional pattern to follow? the old cash java style used to end all comments in a period for some reason...